src/site/_grid.scss

```diff
@@ -55,11 +55,11 @@
 
 /* Generates all the rules for the grid columns up to the given amount of column */
 @mixin generate-columns($breakPoint, $columnTotal, $column: 1) {
-  @if ($column <= $columnTotal) {
+  @if $column <= $columnTotal {
     & > .aem-GridColumn.aem-GridColumn--#{$breakPoint}--#{$column} {
-      float: left;
+      float: inline-start;
```

Comment:
inline-start not working with float
https://developer.mozilla.org/en-US/docs/Web/CSS/float

Reply:
Can you check if these changes work in browser, since the plugin has 9 lakh weekly downloads, I doubt if there would be such issues with the plugin